bindings/python/docs/source/quickstart.rst

```diff
@@ -76,6 +76,10 @@ Nested data (embedded documents) are also supported::
   from pymongoarrow.api import Schema
   schema = Schema({'_id': int, 'amount': float, 'account': { 'name': str, 'account_number': int}})
 
+Arrays (and nested arrays) are also supported::
+
+  from pymongoarrow.api import Schema
+  schema = Schema({'_id': int, 'amount': float, 'account': list_(int32())})
```

Comment:
There is no account field in the example documents above. Could you open a ticket to update these examples?